### PR TITLE
[config] enforce correct chainID and enable IIP-13 at Quebec height

### PR DESCRIPTION
**Pull request by dustinxie**
# Description
Please include a summary of the changes and the related issue. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes #(issue)

## Type of change
Please delete options that are not relevant.
- [] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [] Code refactor or improvement
- [] Breaking change (fix or feature that would cause a new or changed behavior of existing functionality)
- [] This change requires a documentation update

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration
- [x] make test
- [] fullsync
- [] Other test (please specify)

**Test Configuration**:
- Firmware version:
- Hardware:
- Toolchain:
- SDK:

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: api/coreservice.go [hunk 1]
```diff
@@ -416,11 +416,6 @@ func (core *coreService) SendAction(ctx context.Context, in *iotextypes.Action)
 		return "", err
 	}
 
-	// reject web3 rewarding action if isn't activation feature
-	if err := core.validateWeb3Rewarding(selp); err != nil {
```
Comment (dustinxie) [inline api/coreservice.go:420]: no longer needed
should be removed in (#3852)

File: state/factory/workingset_test.go [hunk 2]
```diff
@@ -368,6 +364,7 @@ func makeTransferAction(t *testing.T, nonce uint64) action.SealedEnvelope {
 		SetGasLimit(tsf.GasLimit()).
 		SetGasPrice(tsf.GasPrice()).
 		SetNonce(nonce).
+		SetChainID(1).
```
Comment (dustinxie) [inline state/factory/workingset_test.go:367]: hash values in test changes due to this

File: state/factory/workingset.go [hunk 3]
```diff
@@ -148,8 +148,10 @@ func (ws *workingSet) runAction(
 		return nil, action.ErrGasLimit
 	}
 	// Reject execution of chainID not equal the node's chainID
-	if err := validateChainID(ctx, elp.ChainID()); err != nil {
-		return nil, err
+	if !action.IsSystemAction(elp) {
```
Comment (envestcc) [inline state/factory/workingset.go:151]: why is the system action not  being checked?

**Review by envestcc — APPROVED** [review]
where does the height name `Quebec` come from?

**Comment by dustinxie** [comment]
> where does the height name `Quebec` come from?

Quebec is one of the 13 provinces of Canada

**Review by Liuhaai — APPROVED** [review]

**Pull request merged**